scrapers/privatecastings.yml

```
@@ -0,0 +1,21 @@
+name: privatecastings
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - privatecastings.com/scene/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1
+      Performers:
+        Name: //ul[@class="scene-models-list"]//a
+      Details: //section/h1[1]
```

This gets the title not the description.
The below seems to work ok eg `https://www.privatecastings.com/scene/suzen-seweet-in-a-hardcore-anal-casting-session/67`
```suggestion
      Details: //div[@class="content-desc"]/div/p
```